plugins/api/src/main/java/org/opennms/poc/plugin/api/ServiceMonitorResponseImpl.java

```diff
@@ -7,9 +7,9 @@
 @Builder
 @Data
 public class ServiceMonitorResponseImpl implements ServiceMonitorResponse{
-    private final Status status;
-    private final String reason;
-    private final DeviceConfig deviceConfig;
+    private Status status;
```

Comment:
Making these non-final will cause lombok to generate a setter for them, even though there is a builder pattern in place for setting the fields. Seems like an anti-pattern to allow on something that should be immutable once it's created.